source/common/router/rds_impl.cc

```diff
@@ -120,21 +123,34 @@ void RdsRouteConfigProviderImpl::registerInitTarget(Init::Manager& init_manager)
   init_manager.registerTarget(*this);
 }
 
+std::string RdsRouteConfigProviderImpl::configAsString() const {
+  std::string json_string;
+  Protobuf::util::MessageToJsonString(route_config_proto_, &json_string);
```

Comment:
See my comment here: https://github.com/lyft/envoy/pull/1517#discussion_r134627775. Can we set the same option? Potentially by just moving that option into the MessageToJsonString function? cc @mrice32 @htuch